src/ucp/core/ucp_ep.h

```diff
@@ -478,6 +479,7 @@ typedef struct {
 
 
 enum {
+    UCP_WIREUP_SA_DATA_AM_ONLY = 1,     /* Address has only AM lane information. */
```

Comment:
This value is needed to fix a problem when the private buffer is too small to send all lanes, so it sends only AM lane. But on the other part it tries to establish a connection on ALL found lanes, and since there's no addresses for rma_bw lanes it fails. With this flag send remote part also initializes backward ep with `UCP_ADDRESS_FLAG_AM_ONLY `